src/components/AssetsPage/container.jsx

```diff
@@ -0,0 +1,25 @@
+import { connect } from 'react-redux';
```

Comment:
This is just pulling the mapStateToProps, mapDispatchToProps, and connect out of AssetsPage/index.jsx into AssestsPage/container.jsx; there's no new code